<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for Paleogenomics and Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
Computational Methods for Paleogenomics and Comparative Genomics
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the Computational Paleogenomics group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> and
<a href="http://www.irmacs.sfu.ca">Interdisciplinary Research in the Mathematical and Computational Sciences (IRMACS) Centre</a>
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses primarily on methodological issues
motivated by questions on the evolution of the architecture
of genomes through genome rearrangements. Currently, our main interest
considers the problem of inferring the architecture of ancestral
genomes (of extinct species) through the comparison of related extant
genomes (of descendants species and close outgroups), using
computational methods inspired from physical mapping. In other words,
we are designing methods, algorithms and softwares to compute
ancestral genomes maps or ancestral genomes scaffolds. Combined with
advances in the reconstruction of ancestral genomic sequences,
ancestral genome maps and scaffolds will lead to a fundamental shift
in evolutionary genomics that will consider in a unified framework
both extant and extinct species genomes.
</p>

<p>Our group works also on connected topics, namely phylogenomics,
gene family evolution, in order to infer the ancestral gene content of
extinct species, and genome rearrangements that aim at understanding
large-scale evolutionary events between related species, both extant
and extinct.
</p>

<p>
Our focus being primarily on methodological, mathematical and
computational issues, our research also includes a large theoretical
component, essentially in discrete mathematics and algorithms design
and analysis.
</p>


<h3>News and announcements</h3>

<ul>
<li>October 2016: our paper "Gene Tree Construction and Correction using SuperTree and Reconciliation" with Aida Ouangraoua, Nadia El-Mabrouk and Manuel Lafond has been accepted for publication in the TCBB issue of <a href="http://www.cs.cityu.edu.hk/~shuaicli/apbc2017/">APBC 2017</a>.
<li>September 2016: our lab was awarded two Bioinformatics grants by Genome Canada to work on Flow Cytometry Data (in collaboration with Ryan Brinkman and Sara Mostafavi) and Pathogens analysis (with Leonid Chindelevitch and Will Hsiao): <a href="http://www.genomecanada.ca/en/news-and-events/news-releases/government-canada-invests-new-genomics-big-data-research-projects">news release</a>.
<li>September 2016: two new members join our lab, Aniket Mane (MSc, Math.) and Stanley Guo (MSc, MADD-Gen, co-supervised with Leonid Chindelevitch).
<li>August 2016: Cedric Chauve gives a talk on our on-going work on mosquito genomes at the BC Cancer Agency Genome Sciences Center.
<li>July 2016: Cedric Chauve gives a keynote talk at the <a href="http://dlt2016.lacim.uqam.ca/">DLT 2016</a> conference, on RNA secondary structures comparison.
<li>June 2016: Yoann Anselmetti presents our work on Anopheles genomes at the conference <a href="https://jobim2016.sciencesconf.org/">JOBIM 2016</a>.
<li>May 2016: the papers "ColorMap: Correcting Long Reads by Mapping Short Reads" and "The gene family-free median of three" have been accepted respectively into <a href="http://www.eccb2016.org/">ECCB 2016</a> and <a href="http://conferences.au.dk/algo16/wabi/">WABI 2016</a>.
<li>May 2016: Jonathan Almeida (University of Brasilia) joins the lab as a MITACS GlobaLink intern to work on the software ANGES and on flow cytometry bioinformatics.
<li>May 2016: Sean La (SFU) joins the lab as an NSERC Undergraduate Summer Research Assistant to work on PacBio loong reads correction and tuberculosis evolution.
<li>May 2016: Markus Lux, from the <a href="http://www.didy.uni-bielefeld.de/">Didy program</a> at Bielefeld University joins our group to work on flow cytometry, with Ryan Brinkman (BC Cancer Agency). 
</ul>



</div>
</body>
</html>
