<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<!img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
Computational Methods for Paleogenomics and Comparative Genomics
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the "Computational Methods for Paleogenomics and Comparative Genomics" group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> 
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses currently on various problems in <b>computational biology</b>, with a strong focus on 
<b>comparative genomics</b> and <b>cancer genomics</b>.
We are working on various problems related to phylogenomics (the construction and analysis of 
gene families and gene trees), genome rearrangements and the reconstruction of ancestral gene orders,
pathogen genomics and cancer genomics. 
We are also working on machine-learning algorithms for the analysis of large-scale flow cytometry data.
</p>

See our <a href="research.html">Research page</a> for more details.

<h3>News and announcements</h3>
<ul>
<li>April 2023: our paper "PlasBin-flow: A flow-based MILP algorithm for plasmid contigs binning" has been accepted at <a href="https://www.iscb.org/ismbeccb2023">ISMB/ECCB 2023</a>.
<li>October 2022: our paper "Freddie: Annotation-independent Detection and Discovery of Transcriptomic Alternative Splicing Isoforms Using Long-read Sequencing" has been accepted in <a href="https://academic.oup.com/nar">Nucleic Acids Research</a>.
<li>June 2022: our paper " Fast and accurate matching of cellular barcodes across short- and long-reads of single-cell RNA-seq experiments" has been published in <a href="https://www.cell.com/iscience/home">iScience</a>.
<li>May 2022: our paper "plASgraph - using graph neural networks to detect plasmid contigs from an assembly graph" is available on <a href="https://www.biorxiv.org">biorXiv</a>.
<li>May 2022: our paper "A Mixed Integer Linear Programming Algorithm for Plasmid Binning" has been presented at <a href="https://recombcg2022.usask.ca/">RECOMB â€“ Comparative Genomics 2022</a> and published in <a href="https://www.springer.com/series/5381">Lecture Notes in Bioinformatics</a>.
<li>May 2022: our paper " Fast and accurate matching of cellular barcodes across short- and long-reads of single-cell RNA-seq experiments" has been presented at <a href="https://recomb2022.net/recomb-seq/">RECOMB-Seq 2022</a>.</ul>
</ul>
  
</div>
</body>
</html>
