<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<!img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
Computational Methods for Paleogenomics and Comparative Genomics
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the "Computational Methods for Paleogenomics and Comparative Genomics" group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> 
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses currently on various problems in <b>comparative genomics</b>.
We are working on various problems related to phylogenomics (the construction and analysis of 
gene families and gene trees), genome rearrangements and the reconstruction of ancestral gene orders,
pathogen genomics (genotyping, epidemiologic clustering) and cancer genomics. We are also working
on machine-learning algorithms for the analysis of large-scale flow cytometry data.
</p>

See our <a href="research.html">Research page</a> for more details.

<h3>News and announcements</h3>
<ul>
<li>July 2018: the papers "Detecting introgression in Anopheles mosquito genomes using a reconciliation-based approach" and "The rooted SCJ median with single gene duplications" have been accepted at <a href="https://recombcg2018.usherbrooke.ca/">RECOMB-CG 2018</a>.
<li>June 2018: The paper "PRINCE: Accurate Approximation of the Copy Number of Tandem Repeats" has been accepted at <a href="http://algo2018.hiit.fi/wabi/">WABI 2018</a>.  
<li>May 2018: The paper "Phylogenetic signal from rearrangements in 18 Anopheles species by joint scaffolding extant and ancestral genomes" has been published in <a href="https://bmcgenomics.biomedcentral.com/">BMC Genomics</a>.
<li>May 2018: Robert Mueller, from Bielefeld, joins the lab for 6 months to work on plasmids assembly.
<li>April 2018: The paper "Beaver Fever: Whole-Genome Characterization of Waterborne Outbreak and Sporadic Isolates To Study the Zoonotic Transmission of Giardiasis" has been published in <a href="http://msphere.asm.org/">mSphere</a>.  
<li>March 2018: Our paper "Scaffolding of Ancient Contigs and Ancestral Reconstruction in a Phylogenetic Framework" has been accepted in <a href="https://www.computer.org/web/tcbb">IEEE/ACM TCBB</a>.
<li>March 2018: Our papers "Deconvoluting the diversity of within-host pathogen strains in a Multi-Locus Sequence Typing framework" and "Alignment-free Clustering of Barcode (UMI) Tagged DNA Molecules" have been accepted for oral presentation at <a href="http://recomb2018.fr/recomb-seq/">RECOMB-seq 2018</a>.
<li>February 2018: Our paper "flowLearn: Fast and precise identification and quality checking of cell populations in flow cytometry" has been accepted in <a href="https://academic.oup.com/bioinformatics">Bioinformatics</a>.
<li>January 2018:  Our paper "MentaLiST - A fast MLST caller for large MLST schemes" has been published by <a href="http://mgen.microbiologyresearch.org/content/journal/mgen">Microbial Genomics</a>.
</ul>

</div>
</body>
</html>
