<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for Paleogenomics and Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<!img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
Computational Methods for Paleogenomics and Comparative Genomics
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the "Computational Methods for Paleogenomics and Comparative Genomics" group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> 
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses primarily on computational
approaches to understand the evolution of the architecture of genomes
through genome rearrangements. Currently, our main interest considers
the problem of inferring the architecture of ancestral genomes (of
extinct species) through the comparison of related extant genomes (of
descendants species and close outgroups). We also work on connected
topics, namely phylogenomics, gene family evolution, in order to infer
the ancestral gene content of extinct species, and genome
rearrangements that aim at understanding large-scale evolutionary
events between related species, both extant and extinct.
</p>

See our <a href="research.html">Research page</a> for more details.


<h3>News and announcements</h3>

<ul>
<li>December 2017: Our <a href="">CANSSI</a> Collaborative Research Team proposal "Statistical methods for challenging problems in public health microbiology", lead by Leonid Chnidelevitch and Alexandre Bouchard-C&ocirc;t&eacute;  has been accepted.has been accepted.
<li>December 2017: Release of the code for the SCJ-TD-FD genome rearrangement distance described in our <a href="https://doi.org/10.1007/978-3-319-67979-2_2">RECOMB-CG 2017 paper</a>: <a href="https://github.com/cchauve/SCJ-TD-FD">SCJ-TD-FD</a>.
<li>December 2017: Our paper "MentaLiST - A fast MLST caller for large MLST schemes" has been accepted by <a href="http://mgen.microbiologyresearch.org/content/journal/mgen">Microbial Genomics</a>.
<li>November 2017: Our paper "MaxTiC: Fast Ranking Of A Phylogenetic Tree By Maximum Time Consistency With Lateral Gene Transfers" has now been recommended by <a href="http://dx.doi.org/10.24072/pci.evolbiol.100037">PCI Evol Biol</a>.
<li>October 2017: Our paper "Phylogenetic signal from rearrangements in 18 Anopheles species by joint scaffolding extant and ancestral genomes" has been accepted at <a href="http://apbc2018.bio.keio.ac.jp/">APBC 2018</a>.
<li>September 2017: Congratulations to Fang Zhang who defended successfuly his MSc thesis "Computational exploration of transmission and acquisition of drug-resistant tuberculosis".
<li>September 2017: Two new students join our group. Baraa Orabi (co-supervised with Faraz Hach) and Hooman Zabeti (co-supervised with leonid Chindelevitch and Tamon Stephen)
<li>August 2017: Congratulations to ALice Yue who defended successfuly her MSc thesis "Feature-based Comparison of Flow Cytometry Data".
<li>August 2017: Our preprint "MentaLiST - A fast MLST caller for large MLST schemes" is available on <a href="http://www.biorxiv.org/">biorXiv</a>.  
<li>July 2017: Our paper "LRCstats, a tool for evaluating long reads correction methods" has been accepted in <a href="https://academic.oup.com/bioinformatics">Bioinformatics</a>
<li>July 2017: Our paper "Constructing a Consensus Phylogeny from a Leaf-Removal Distance" has been accepted in <a href="http://pages.di.unipi.it/spire2017/">SPIRE 2017</a>.
<li>July 2017: The final version of our paper on scaffolding ancient Yersini pestis genomes has been published in <a href="<br>Final version published in 2017 in <a href="http://dx.doi.org/10.1099/mgen.0.000123">Microbial Genomics</a>.  
<li>May 2017: The full length version of our paper "The gene family-free median of three" has been published in <a href ="https://almob.biomedcentral.com/articles/10.1186/s13015-017-0106-z">Algorithms for Molecular Biology</a>
</ul>



</div>
</body>
</html>
