<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<!img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
Computational Methods for Paleogenomics and Comparative Genomics
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the "Computational Methods for Paleogenomics and Comparative Genomics" group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> 
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses currently on various problems in <b>comparative genomics</b>.
We are working on various problems related to phylogenomics (the construction and analysis of 
gene families and gene trees), genome rearrangements and the reconstruction of ancestral gene orders,
pathogen genomics (genotyping, epidemiologic clustering) and cancer genomics. We are also working
on machine-learning algorithms for the analysis of large-scale flow cytometry data.
</p>

See our <a href="research.html">Research page</a> for more details.

<h3>News and announcements</h3>
<ul>
<li>September 2020: Cedric gives a presentation on long reads transcriptomics at the <a href="https://www.pangenome.eu/news-event-pangaia/computational-pangenomics-algorithms-applications-eccb-2020/">ECCB 2020 PANGAIA workshop</a>.
<li>September 2020: our paper "A graph-theoretic barcode ordering model for linked-reads" is now available in the < a href="https://drops.dagstuhl.de/opus/volltexte/2020/12800/">proceedings of WABI 2020</a>.
<li>August 2020: our paper "HASLR: Fast Hybrid Assembly of Long Reads" has been published in <a href="https://www.cell.com/iscience/home">iScience/home</a>.
<li>June 2020: the paper "A graph-theoretic barcode ordering model for linked-reads" has been accepted for presentation at <a href="http://algo2020.di.unipi.it/WABI2020/index.html">WABI 2020</a>.
<li>May 2020: our paper "The Distance and Median Problems in the Single-Cut-or-Join model with Single-Gene Duplications" has been published in <a href="https://almob.biomedcentral.com">Algorithms for Molecular Biology</a>.
<li>April 2020: The open access book <a href="https://hal.inria.fr/PGE/">Phylogenetics in the Genomic Era</a>, that includes our chapter "Ancestral Genome Organization as a Diagnosis Tool forPhylogenomics" is now published.
<li>March 2020: Ehsan Haghshenas successfully defended his PhD thesis; congrats Ehsan.
<li>February 2020: our paper "HASLR: Fast Hybrid Assembly of Long Reads" has been accepted at <a href="https://www.recomb2020.org/recomb-seq-satellites">RECOMB-Seq 2020</a>
<li>January 2020: Cedric joins the Editorial Board of <a href="https://academic.oup.com/bioinformatics/">Bioinformaics</a>.
<li>January 2020: our paper on the improving Anopheles genomes assemblies has been published in <a href="https://bmcbiol.biomedcentral.com/">BMC Biology</a>.  
</ul>

</div>
</body>
</html>
