<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for Paleogenomics and Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
Computational Methods for Paleogenomics and Comparative Genomics
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the Computational Paleogenomics group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> and
<a href="http://www.irmacs.sfu.ca">Interdisciplinary Research in the Mathematical and Computational Sciences (IRMACS) Centre</a>
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses primarily on computational
approaches to understand the evolution of the architecture of genomes
through genome rearrangements. Currently, our main interest considers
the problem of inferring the architecture of ancestral genomes (of
extinct species) through the comparison of related extant genomes (of
descendants species and close outgroups). We also work on connected
topics, namely phylogenomics, gene family evolution, in order to infer
the ancestral gene content of extinct species, and genome
rearrangements that aim at understanding large-scale evolutionary
events between related species, both extant and extinct.
</p>

See our <a href="research.html">Research page</a> for more details.


<h3>News and announcements</h3>

<ul>
<li>May 2017: the full length version of our paper "The gene family-free median of three" has been published in <a href ="https://almob.biomedcentral.com/articles/10.1186/s13015-017-0106-z">Algorithms for Molecular Biology</a>
<li>April 2017: The papers "MaxTiC: Fast Ranking Of A Phylogenetic Tree By Maximum Time Consistency With Lateral Gene Transfers" (<a href="http://biorxiv.org/content/early/2017/04/14/127548.full.pdf">biorXiv</a>) and "DeCoSTAR: Reconstructing the ancestral organization of genes or genomes using reconciled phylogenies" (<a href="https://academic.oup.com/gbe/article-lookup/doi/10.1093/gbe/evx069">Genome Biolgy and Evolution</a>) are now available.
<li>April 2017: Hua-Ting Yao (Polytechnique) and Etienne Lassale (ENS Cachan) join our PathoGiST project. Welcome.
<li>February 2017: Cedric and Pedro give talks at the workshop <a href="http://www.birs.ca/events/2017/5-day-workshops/17w5104">Mathematical Approaches to Evolutionary Trees and Networks</a>
<li>January 2017: Pedro Feijao joins the lab as a post-doctoral fellow, co-supervised with Leonid Chindelevitch and Will Hsiao, to work on computational epidemiology (Genome Canada project).
<li>December 2016: the full length version of our paper "The SCJ Small Parsimony Problem for Weighted Gene Adjacencies" has been accepted in <a href="http://ieeexplore.ieee.org/document/7837680/">IEEE/ACM TCBB</a>.
<li>December 2016: Nina Luhmann successfuly defends her PhD thesis at Bielefeld University!
<li>November 2016: welcome to post-doctoral fellow Stepan Grynek who will work on flow cytometry bioinfomatics (genome Canada project, with Ryan Brinkman and Sara Mostafavi).
<li>October 2016:  our paper "Gene Tree Construction and Correction using SuperTree and Reconciliation" with Aida Ouangraoua, Nadia El-Mabrouk and Manuel Lafond has been accepted for publication in the TCBB issue of <a href="http://www.cs.cityu.edu.hk/~shuaicli/apbc2017/">APBC 2017</a>.
<li>September 2016: We present our on-going work on improving the genome assemblies of <i>Anopheles</i> mosquitoes genomes at the <a href="http://ice2016orlando.org/">International Congress of Entemology</a>.
<li>September 2016: our lab was awarded two Bioinformatics grants by Genome Canada to work on Flow Cytometry Data (in collaboration with Ryan Brinkman and Sara Mostafavi) and Pathogens analysis (with Leonid Chindelevitch and Will Hsiao): <a href="http://www.genomecanada.ca/en/news-and-events/news-releases/government-canada-invests-new-genomics-big-data-research-projects">news release</a>.
<li>September 2016: two new members join our lab, Aniket Mane (MSc, Math.) and Stanley Guo (MSc, MADD-Gen, co-supervised with Leonid Chindelevitch).
</ul>



</div>
</body>
</html>
