<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational (paleo)genomics Group at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<!img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
Computational Methods for Paleogenomics and Comparative Genomics
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">HOME -
  <a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Welcome to the "Computational Methods for Paleogenomics and Comparative Genomics" group of
<a href="http://www.cecm.sfu.ca/~cchauve">Cedric Chauve</a>,
<a href="http://www.math.sfu.ca">Department of Mathematics</a> 
at 
<a href="http://www.sfu.ca/">Simon Fraser University</a>.</p>

<p>
The research of our group focuses currently on various problems in <b>computational biology</b>, with a strong focus on 
<b>comparative genomics</b> and <b>cancer genomics</b>.
We are working on various problems related to phylogenomics (the construction and analysis of 
gene families and gene trees), genome rearrangements and the reconstruction of ancestral gene orders,
pathogen genomics (genotyping, epidemiologic clustering, plasmids assembly) and cancer genomics. 
We are also working on machine-learning algorithms for the analysis of large-scale flow cytometry data.
</p>

See our <a href="research.html">Research page</a> for more details.

<h3>News and announcements</h3>
<ul>
<li>December 2021: the paper "Genion, an accurate tool to detect gene fusion from long transcriptomics reads" has been accepted in <a href="https://bmcgenomics.biomedcentral.com">BMC Genomics</a>.
<li>November 2021: the paper "Automated identification of maximal differential cell populations in flow cytometry data" has been published in <a href="https://onlinelibrary.wiley.com/journal/15524930">Cytometry A</a>.
<li>November 2021: our work in progress "Gating flow cytometry samples using few-shot image segmentation" has been presented as a spotlight at <a href="https://sites.google.com/cs.washington.edu/mlcb2021">MLCB 2021</a>.
<li>July 2021: the paper "Small Parsimony for Natural Genomes in the DCJ-Indel Model" has been accepted to the <a href="https://recombcg2021.wordpress.com/">RECOMB â€“ Comparative Genomics 2021</a> conference.
<li>June 2021: the paper "Refined upper bounds on the size of the condensed neighbourhood of sequences" has been accepted to the <a href="http://www.stringology.org/event/">The Prague Stringology Conference 2021</a>.
<li>January 2021: our paper "Freddie: Annotation-independent Detection and Discovery of Transcriptomic Alternative Splicing Isoforms" on long reads transcriptomics, to be presented at <a href="https://www.recomb2021.org/">RECOMB 2021</a> is available on <a href="https://www.biorxiv.org/content/10.1101/2021.01.20.427493v1.abstract">bioRxiv<a/>.
<li>September 2020: our paper "A graph-theoretic barcode ordering model for linked-reads" is now available in the <a href="https://drops.dagstuhl.de/opus/volltexte/2020/12800/">proceedings of WABI 2020</a>.
<li>April 2020: The open access book <a href="https://hal.inria.fr/PGE/">Phylogenetics in the Genomic Era</a>, that includes our chapter "Ancestral Genome Organization as a Diagnosis Tool forPhylogenomics" is now published.
</ul>

</div>
</body>
</html>
