<!doctype HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational paleogenomics Lab at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for Paleogenomics and Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
Computational Methods for Paleogenomics and Comparative Genomics
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">
<a href="index.html">HOME</a> -
<a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
PUBLICATIONS -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<p>Selected Publications. A complete list, including preliminary
versions and posters is
available <a href="http://www.cecm.sfu.ca/~cchauve/publications.html">here</a>

<h1>2017</h1>
<ul> 
<li><a style="color: #800000" href="https://link-springer-com.proxy.lib.sfu.ca/content/pdf/10.1007%2F978-3-319-67979-2_2.pdf">A tractable variant of the Single Cut or Join distance with duplicated genes.</a>
<br><b>P. Feijao, A. Mane, C. Chauve</b>, <a href="http://www.crg.eu/en/event/15th-recomb-comparative-genomics-satellite-workshop">RECOMB-CG 2017</a>.
<a href="https://doi.org/10.1007/978-3-319-67979-2_2">LNCS/LNBI 10562:14-30</a>, 2017.  
<li><a style="color: #800000" href="http://www.biorxiv.org/content/early/2017/08/06/172858.full.pdf">MentaLiST - A fast MLST caller for large MLST schemes</a>.
<br><b>P. Feijao, H. Yao, D. Fornika,  J. Gardy,  W. Hsiao,  C. Chauve, L. Chindelevitch</b>, <a href="http://www.biorxiv.org/content/early/2017/08/06/172858">biorXiv 172858</a>, 2017.
<li><a style="color: #800000" href="https://academic.oup.com/bioinformatics/article-pdf/doi/10.1093/bioinformatics/btx489/19397317/btx489.pdf">LRCstats, a tool for evaluating long reads correction methods</a>.
<br><b>S. La, E. Haghshenas, C. Chauve</b>,  <a href="https://doi.org/10.1093/bioinformatics/btx489">Bioinformatics, advance access</a>, 2017.
<li><a style="color: #800000" href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-017-1267-2?site=genomebiology.biomedcentral.com">ReMixT: clone-specific genomic structure estimation in cancer
</a>.
<br><b>A. W. McPherson, A. Roth, G. Ha, C. Chauve, A. Steif, C. P. E. de Souza, P. Eirew, A. Bouchard-Côté, S. Aparicio, S. C. Sahinalp and S. P. Shah</b>, <a href="https://doi.org/10.1186/s13059-017-1267-2">Genome Biology 18:140</a>, 2017.
<li><a style="color: #800000" href="http://mgen.microbiologyresearch.org/deliver/fulltext/mgen/mgen.000123.zip/mgen000123.pdf?itemId=/content/journal/mgen/10.1099/mgen.0.000123.v1&mimeType=pdf">Comparative scaffolding and gap filling of ancient bacterial genomes applied to two ancient Yersinia pestis genomes</a>.
<br><b>N. Luhmann, D. Doerr, C. Chauve</b>, <a href="http://dx.doi.org/10.1099/mgen.0.000123">Microbial Genomics 2017:3</a>, 2017.
<li><a style="color: #800000" href="http://download.springer.com/static/pdf/826/art%253A10.1186%252Fs13015-017-0106-z.pdf">The gene family-free median of three</a>.
<br><b>D. Doerr, M. Balaban, P. Feijao, C. Chauve</b>, <a href="https://almob.biomedcentral.com/articles/10.1186/s13015-017-0106-z">Algorithms for Molecular Biology 12:14</a>, 2017.
<br>Preliminary version (2016) in the proceedings of <a href="http://conferences.au.dk/algo16/wabi/">WABI 2016</a>. 
<li><a style="color: #800000" href="https://link-springer-com.proxy.lib.sfu.ca/content/pdf/10.1007%2F978-3-319-67428-5_12.pdf">Constructing a Consensus Phylogeny from a Leaf-Removal Distance</a>
<br><b>C. Chauve, M. Jones, M. Lafond, C. Scornavacca, M. Weller</b>, <a href="http://pages.di.unipi.it/spire2017/">SPIRE 2017</a>. <a href="https://dx.doi.org/10.1007/978-3-319-67428-5_12">LNCS 10508:129-143</a>. 2017.
<br><b>C. Chauve, M. Jones, M. Lafond, C. Scornavacca, M. Weller</b>, <a href="https://arxiv.org/abs/1705.05295">arXiv:1705.05295</a>. 2017.
<li><a style="color: #800000" href="https://academic.oup.com/gbe/article-pdf/doi/10.1093/gbe/evx069/11474030/evx069.pdf">DeCoSTAR: Reconstructing the ancestral organization of genes or genomes using reconciled phylogenies</a>.
<br><b>W. Duchemin, Y. Anselmetti, M. Patterson, Y. Ponty, S. B&eacute;rard, C. Chauve, C. Scornavacca, V. Daubin, E. Tannier</b>, 
<a href="https://academic.oup.com/gbe/article-lookup/doi/10.1093/gbe/evx069">Genome Biology and Evolution 9(5):1312-1319</a>, 2017.
<li><a style="color: #800000" href="http://biorxiv.org/content/early/2017/04/14/127548.full.pdf">MaxTiC: Fast Ranking Of A Phylogenetic Tree By Maximum Time Consistency With Lateral Gene Transfers</a>.
<br><b>C. Chauve, A. Rafiey, A. Davin, C. Scornavacca, P. Veber, B. Boussau, G. Szollosi, V. Daubin, E. Tannier</b>, <a href="https://doi.org/10.1101/127548">biorXiv 127548</a>.
<li><a style="color: #800000" href="https://academic.oup.com/bib/article-pdf/18/2/306/11008711/bbw022.pdf">The Bralibase dent: A tale of benchmark design and interpretation</a>.
<br><b>B. Loews, C. Chauve, Y. Ponty, R. Giegerich</b>, <a href="https://academic.oup.com/bib/article-lookup/doi/10.1093/bib/bbw022">Briefings  in Bioinformatics 18(2):306-311</a>, 2017.
<li><a style="color: #800000" href="http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7837680">The SCJ Small Parsimony Problem for Weighted Gene Adjacencies</a>
<br><b>N. Luhmann, M. Lafond, A. Th&eacute;venin, A. Ouangraoua, R. Wittler, C. Chauve</b>, <a href="https://doi.org/10.1109/TCBB.2017.2661761">IEEE/ACM TCBB Early Access</a>, 2017.
<br>Preliminary version (2016) in the proceedings of <a href="http://grid.cs.gsu.edu/isbra16/">ISBRA 2016</a>.
<li><a style="color: #800000" href="https://arxiv.org/pdf/1610.05068v2">Gene Tree Construction and Correction using SuperTree and Reconciliation</a>
<br><b>M. Lafond, C. Chauve, A. Ouangraoua, N. El-Mabrouk</b>, <a href="https://doi.org/10.1109/TCBB.2017.2720581">IEEE/ACM TCBB Early Access</a>, 2017. Proceedings of <a href="http://www.cs.cityu.edu.hk/~shuaicli/apbc2017/">APBC 2017</a>.
<li>(From a previous life as a combinatorialist) <a style="color: #800000" href="http://epubs.siam.org/doi/pdf/10.1137/1.9781611974775.3">An Exact Enumeration of Distance-Hereditary Graphs</a>.
<br><b>C. Chauve, E. Fusy, J. Lumbroso</b>, proceedings of <a href="http://dx.doi.org/10.1137/1.9781611974775.3">ANALCO 2017, pp. 31-45</a>.
<li><a style="color: #800000" href="http://doi.ieeecomputersociety.org/10.1109/TCBB.2016.2528239">Algorithms and complexity results for genome mapping problems</a>.
<br><b>A. Rajaraman, J.P.P. Zanetti, J. Manuch, C. Chauve</b>, <a href="http://doi.ieeecomputersociety.org/10.1109/TCBB.2016.2528239">IEEE/ACM TCBB 14(2):418-430</a>. 2017.
</ul>
  
<h1>2016</h1>
<ul>
<li><a style="color: #800000" href="http://bioinformatics.oxfordjournals.org/content/32/17/i545.full.pdf">CoLoRMap: Correcting Long Reads by Mapping short reads</a>
<br><b>E. Haghshenas, F. Hach, S.C. Sahinalp, C. Chauve</b>, <a href="http://bioinformatics.oxfordjournals.org/content/32/17/i545">Bioinformatics 32(17):i545-i551</a>, 2016.
<li><a style="color: #800000" href="http://biorxiv.org/content/early/2016/09/05/073445.full.pdf">Improved assemblies and comparison of two ancient Yersinia pestis genomes</a>
<br><b>N. Luhmann, D. Doerr, C. Chauve</b>, <a href="http://biorxiv.org/content/early/2016/09/05/073445">bioRxiv:073445</a>. 2016.
<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007%2F978-3-319-43681-4_9.pdf">The Gene Family-Free Median of Three</a>
<br><b>D. Doerr, P. Feijao, M. Balaban, C. Chauve</b>, <a href="http://conferences.au.dk/algo16/wabi/">WABI 2016</a>, <a href="http://dx.doi.org/10.1007/978-3-319-43681-4_9">LNCS/LNBI 9838:102-120</a>, 2016.
<br><b>D. Doerr, P. Feijao, M. Balaban, C. Chauve</b>, <a href="https://arxiv.org/abs/1606.05910">arXiv:1606.05910v1 [cs.DS]</a>, 2016.
<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007%2F978-3-319-38782-6_17.pdf">The SCJ small parsimony problem for weighted gene adjacencies</a>.
<br><b>N. Luhmann, A. Th&eacute;venin, A. Ouangraoua, R. Wittler, C. Chauve</b>, <a href="http://grid.cs.gsu.edu/isbra16/">ISBRA 2016</a>, <a href="http://dx.doi.org/10.1007/978-3-319-38782-6_17">LNCS/LNBI 9683:200-210</a>, 2016.
<br><b>N. Luhmann, A. Th&eacute;venin, A. Ouangraoua, R. Wittler, C. Chauve</b>, <a href="http://arxiv.org/abs/1603.08819">arXiv:1603.08819 [cs.DS]</a>, 2016.
<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007%2F978-3-319-38827-4_5.pdf">Counting, generating and sampling tree alignments</a>.
<br><b>J. Courtiel, C. Chauve, Y. Ponty</b>,  <a href="http://grammars.grlmc.com/AlCoB2016/">AlCoB 2016</a>, <a href="http://dx.doi.org/10.1007/978-3-319-38827-4_5">LNCS/LNBI 9702:53-64</a>, 2016.
<br><b>J. Courtiel, C. Chauve, Y. Ponty</b>,  <a href="https://hal.inria.fr/hal-01154030v2">HAL 01154030v2</a>, 2016.
<li><a style="color: #800000" href="http://bioinformatics.oxfordjournals.org/content/32/13/2056.full.pdf">ecceTERA: Comprehensive gene tree-species tree reconciliation using parsimony</a>.
<br><b>E. Jacox, C. Chauve, G. Szollosi, Y. Ponty, C. Scornavacca</b>, <a href="http://dx.doi.org/ 10.1093/bioinformatics/btw105">Bioinformatics 32(17):2056-2058</a>, 2016.
</ul>


<h1>2015</h1>
<ul>
<li>
<a style="color: #800000" href="http://www.biomedcentral.com/content/pdf/1471-2105-16-S19-S6.pdf">Evolution of genes neighborhood within reconciled phylogenies: an ensemble approach</a>. 
<br><b>C. Chauve, Y. Ponty, J.P.P. Zanetti</b>, <a href="http://www.biomedcentral.com/1471-2105/16/S19/S6">BMC Bioinformatics 2015, 16(Suppl 19):S6</a>, 2015.
<br>Preprint: <a href="http://biorxiv.org/cgi/content/abstract/026310v1">bioRxiv:026310</a>, 2015.
<br>Preliminary version (2014) in the proceedings of <a href="http://www.iscb.org/iscb-latinamerica2014-bsb-symposium">BSB 2014</a>.
<li><a style="color: #800000" href="http://www.biomedcentral.com/content/pdf/1471-2164-16-S10-S11.pdf">Ancestral gene synteny reconstruction improves extant species scaffolding</a>.
<br><b>Y. Anselmetti, V. Berry, C. Chauve, A. Chateau, E. Tannier, S. Berard</b>, <a href="http://biorxiv.org/content/early/2015/07/23/023085.abstract">biorXiv:023085</a>, 2015.
<br><b>Y. Anselmetti, V. Berry, C. Chauve, A. Chateau, E. Tannier, S. Berard</b>, <a href="https://applbio.biologie.uni-frankfurt.de/recombcg2015/">RECOMB-CG 2015</a>, <a href="http://www.biomedcentral.com/1471-2164/16/S10/S11">BMC Genomics 16(Suppl 10):S11</a>, 2015.
<li><a style="color: #800000" href="http://arxiv.org/pdf/1506.07458.pdf">Chaining fragments in sequences: to sweep or not</a>.
<br><b>J. Allali, L. Bourgeade, C. Chauve</b>, <a href="http://arxiv.org/abs/1506.07458">arXiv:1506.07458</a>, 2015.
<br><b>J. Allali, L. Bourgeade, C. Chauve</b>, <a href="http://www.dcs.kcl.ac.uk/events/spire2015/">SPIRE 2015</a>, <a href="http://link.springer.com/chapter/10.1007%2F978-3-319-23826-5_12">LNCS 9309:116-123</a>, 2015. 

<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007%2F978-3-319-19048-8_22.pdf">Assessing the robustness of parsimonious predictions for gene neighborhoods from reconciled phylogenies</a>.
<br><b>A. Rajaraman, C. Chauve, Y. Ponty</b>, <a href="http://www.cs.gsu.edu/isbra15/">ISBRA 2015</a>, <a href="http://link.springer.com/chapter/10.1007/978-3-319-19048-8_22">LNCS/LNBI 9096:260-271</a>, 2015.
<br>Preprint: <a href="http://arxiv.org/abs/1503.05292">arXiv:1503.05292</a>, 2015.
<li><a style="color: #800000" href="http://online.liebertpub.com/doi/pdf/10.1089/cmb.2014.0283">Chaining Sequence/Structure Seeds for Computing RNA Similarity</a>.
<br><b>L. Bourgeade, C. Chauve, J. Allali.</b>, <a href="http://online.liebertpub.com/doi/10.1089/cmb.2014.0283">J. Comput. Biol. 22(3):205-217</a>, 2015.
<br>Preliminary version (2014) in the proceedings of <a href="http://cmsr14.cs.mcgill.ca/">CMSR 2014</a>.  
<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007/978-3-319-16706-0_25.pdf">Joint inference of genome structure and content in heterogeneous tumor samples</a>. 
<br><b>A. McPherson, A. Roth, C. Chauve, S.C. Sahinalp</b>, <a href="http://www.recomb2015.pl/">RECOMB 2015</a>, <a href="http://link.springer.com/chapter/10.1007%2F978-3-319-16706-0_25">LNCS/LNBI 9069:256-258</a>, 2015.
<br>
<b>A. McPherson, A. Roth, G. Ha, S.P. Shah, C. Chauve, S.C. Sahinalp</b>, <a href="http://arxiv.org/abs/1504.03080">arXiv:1504.03080 [q-bio.GN]</a>, 2015.
</ul>

<h1>2014</h1>
<ul>
<li><a style="color: #800000" href="http://www.sciencemag.org/content/347/6217/1258522.full.pdf">Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes </a>.
<br><b>D. E. Neafsey, R. M. Waterhouse, et al</b> <a href="http://www.sciencemag.org/content/347/6217/1258522.abstract">Science 347:1258522</a>, 2015.
<li><a style="color: #800000" href="http://bioinformatics.oxfordjournals.org/content/30/17/i519.full.pdf">Polytomy Refinement for the Correction of Dubious Duplications in Gene Trees</a>.
<br><b>M. Lafond, C. Chauve, R. Dondi, N. El-Mabrouk</b> <a href="http://www.eccb14.org">ECCB 2014</a>, <a href="http://bioinformatics.oxfordjournals.org/content/30/17/i519">Bioinformatics 30:i519-i526</a>, 2014.
<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007%2F978-3-319-12418-6_7.pdf">Evolution of genes neighborhood within reconciled phylogenies: an ensemble approach</a>.
<br><b>C. Chauve, Y. Ponty, J.P.P. Zanetti</b> <a href="http://www.iscb.org/iscb-latinamerica2014-bsb-symposium">BSB 2014</a>, <a href="http://dx.doi.org/10.1007/978-3-319-12418-6_7">LNCS/LNBI 8826:49-56</a>, 2014.
<li><a style="color: #800000" href="http://link.springer.com/content/pdf/10.1007%2F978-3-319-12418-6_17.pdf">Scaffolding of Ancient Contigs and Ancestral Reconstruction in a Phylogenetic Framework</a>.
<br><b>N. Luhmann, C. Chauve, J. Stoye, R. Wittler</b> <a href="http://www.iscb.org/iscb-latinamerica2014-bsb-symposium">BSB 2014</a>, <a href="http://dx.doi.org/10.1007/978-3-319-12418-6_17">LNCS/LNBI 8826:135-143</a>, 2014.
<li><a style="color: #800000" href="http://cmsr14.cs.mcgill.ca/pdf/CMSR14-Article-Bourgeade.pdf">Chaining Sequence/Structure Seeds for Computing RNA Similarity</a>.
<br><b>L. Bourgeade, C. Chauve, J. Allali.</b> <a href="http://cmsr14.cs.mcgill.ca/">CMSR 2014</a>, pp. 1-12. 2014.
</ul>


<h1>2013</h1>
<ul>

<li><a style="color: #800000" href="http://dx.doi.org/10.1007/978-1-4471-5298-9">Models and Algorithms for Genome Evolution</a>.
<br><b>C. Chauve, N. El-Mabrouk, E Tannier (eds).</b> <a href="http://link.springer.com/bookseries/5769">Computational Biology</a>. 2013.

<li><a style="color: #800000" href="http://dx.doi.org/10.1007/978-1-4471-5298-9_4">Duplication, Rearrangement and Reconciliation: A Follow-Up 13 Years Later</a>.
<br><b>C. Chauve, N. El-Mabrouk, L. Gu&eacute;guen, M. Semeria, E Tannier.</b> <a href="http://dx.doi.org/10.1007/978-1-4471-5298-9">Models and Algorithms for Genome Evolution</a>, <a href="http://link.springer.com/bookseries/5769">Computational Biology</a>, 19:47-62. 2013.

<li><a style="color: #800000" href="http://dx.doi.org/10.1007/978-1-4471-5298-9_13">The Potential of Family-Free Genome Comparison</a>.
<br><b>M. Braga, C. Chauve, D. Doerr, K. Jahn, J. Stoye, A. Th&eacute;venin, R. Wittler.</b>  <a href="http://dx.doi.org/10.1007/978-1-4471-5298-9">Models and Algorithms for Genome Evolution</a>, <a href="http://link.springer.com/bookseries/5769">Computational Biology</a>, 19:287-307. 2013.


<li> <a style="color: #800000" href="http://bioinformatics.oxfordjournals.org/content/29/23/2987.abstract.html">FPSAC: Fast Phylogenetic Scaffolding of Ancient Contigs</a>.
<br><b>A. Rajaraman, E. Tannier,  C. Chauve.</b> <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics, 29:2987-2994</a>. 2013.

<li> <a style="color: #800000" href="http://arxiv.org/pdf/1307.7642v1">The genome of the medieval Black Death agent (extended abstract)</a>.
<br><b>A. Rajaraman, E. Tannier,  C. Chauve.</b> <a href="http://arxiv.org/abs/1307.7642">arXiv:1307.7642 [q-bio.GN]</a>. 2013.
<br>Presented as a talk at <a href="https://colloque.inra.fr/jobim2013_eng/">JOBIM 2013</a>.

<li> <a style="color: #800000" href="http://arxiv.org/pdf/1306.4353v2">Hypergraph covering problems motivated by genome assembly questions</a>.  
<br><b>C. Chauve, M. Patterson, A. Rajaraman.</b> <a href="http://arxiv.org/abs/1306.4353">arXiV:1306.4353v2 [cs.DS]</a> and <a href="http://iwoca2013.colloques.univ-rouen.fr/">IWOCA 2013</a>, <a href="http://dx.doi.org/10.1007/978-3-642-45278-9_37">LNCS 8288:428-432</a>. 2013.
</ul>

<h1>2012</h1>
<ul>

<li><a style="color: #800000"  href="http://www.biomedcentral.com/1471-2105/13/S19/S11">Linearization of ancestral multichromosomal genomes</a>. 
<br><b>J. Manuch, M. Patterson, R. Wittler, C. Chauve, E. Tannier.</b> <a href="http://www.uff.br/recombcg/">RECOMB-CG 2012</a>, <a href="http://www.biomedcentral.com/1471-2105/13/S19/S11">BMC Bioinformatics 13(Suppl. 19):S11</a>.

<li><a style="color: #800000" href="http://bioinformatics.oxfordjournals.org/content/28/18/2388.full.pdf+html">ANGES: reconstructing ANcestral GEnomeS maps</a>.
<br><b>B.R. Jones, A. Rajaraman, E. Tannier, C. Chauve.</b> <a href="http://bioinformatics.oxfordjournals.org/content/28/18/2388">Bioinformatics, 28:2388-2390</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications/1109.0562v1.pdf">A tight bound on the length of odd cycles in the incompatibility graph of a non-C1P matrix</a>. 
<br><b>M. Malekesmaeili, C. Chauve, T. Stephen. </b> <a href="http://dx.doi.org/10.1016/j.ipl.2012.07.004">IPL 12:799-803</a>.

<li><a style="color: #800000" href="http://www.computer.org/portal/web/csdl/doi/10.1109/TCBB.2011.64">An Efficient Method for Exploring the Space of Gene Tree/Species Tree Reconciliations in a Probabilistic Framework</a>. 
<br><b>J.P. Doyon, S. Hamel, C. Chauve.</b> <a href="http://www.computer.org/portal/web/csdl/doi/10.1109/TCBB.2011.64">IEEE/ACM TCBB 9:26-39</a>.

<li><a style="color: #800000" href="http://dx.doi.org/10.1016/j.dam.2012.03.019">Hardness Results for the Gapped Consecutive-Ones Property</a>. 
<br><b>J. Manuch, M. Patterson, C. Chauve. </b><a href="http://www.journals.elsevier.com/discrete-applied-mathematics/">Discrete Applied Math</a>, 160(18):2760-2768.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//JDA_IWOCA10.pdf">Efficient chaining of seeds in ordered trees</a>.
<br><b>J. Allali, C. Chauve, P. Ferraro, A.-L. Gaillard. </b><a href="http://dx.doi.org/10.1016/j.jda.2011.12.013">J. Discrete Algorithms 14:107-118</a>.

<li><a style="color: #800000" href="http://www.hindawi.com/journals/abi/2012/893048/">BRASERO: A resource for benchmarking RNA secondary structure comparison algorithms<a/>. 
<br><b>J. Allali et al.</b> <a href="http://www.hindawi.com/journals/abi/2012/893048/">Advances in Bioinformatics 2012:893048</a>.
</ul>

<h1>2011</h1>
<ul>
<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//HDR_CEDRIC_CHAUVE_2011.pdf">&Agrave; la recherche des g&eacute;nomes perdus : Mod&egrave;les, M&eacute;thodologies et Algorithmes  pour la Comparaison de G&eacute;nomes</a>. 
<br><b>C. Chauve. </b> Habilitation &agrave; diriger des recherches, <a href="http://www.labri.fr/">LaBRI, University Bordeaux I</a>.

<li><a style="color: #800000" href="http://www.biomedcentral.com/1471-2105/12/S9/S21">Consistency-based detection of potential tumor-specific deletions in matched normal/tumor genomes</a>. 
<br><b>R. Witler, C. Chauve. </b> <a href="http://www.recombcg.org/">RECOMB-CG 2011</a>, <a href="http://www.biomedcentral.com/1471-2105/12/S9/S21">BMC Bioinformatics 12:S21</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//ISMB11.pdf">Mapping ancestral genomes with massive gene loss: a matrix sandwich problem</a>.  
<br><b>H. Gavranovic, C. Chauve, J. Salse, E. Tannier. </b> <a href="www.iscb.org/ismbeccb2011">ISMB/ECCB 2011</a>, <a href="http://bioinformatics.oxfordjournals.org/content/27/13/i257">Bioinformatics 27:i257-i265</a>.

<li><a style="color: #800000" href="http://bioinformatics.oxfordjournals.org/content/27/19/2664">Reconstructing the architecture of the ancestral amniote genome</a>. 
<br><b>A. Ouangraoua, E. Tannier, C. Chauve. </b> <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics 27:2664-2671</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//JCB_RCG10.pdf">A 2-Approximation for the Minimum Duplication Speciation Problem<a/>. 
<br><b>A. Ouangraoua, K. Swenson, C. Chauve. </b><a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2011.0108">J. Comp. Biol. 18:1041-1053</a>.

<li><a  style="color: #800000" href="http://www.springerlink.com/content/m2688m401q06p320/">Branch-and-Bound Approach for Parsimonious Inference of a Species Tree From a Set of Gene Family Trees</a>. 
<br><b>J.P. Doyon, C. Chauve. </b> <a href="http://www.springerlink.com/content/978-1-4419-7045-9/">Software Tools and Algorithms for Biological Systems, chap. 29</a>. <br>

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//DMAA_CPM09.pdf">Average-case analysis of perfect sorting by reversals</a>.  
<br><b>M. Bouvel, C. Chauve, M. Mishna, D. Rossin. </b> <a href="http://www.worldscinet.com/dmaa/03/0303/S1793830911001280.html">Discrete Math. Algorithms Applications 3:369-392</a>.

<li><a style="color: #800000" href="http://arxiv.org/pdf/1111.5872v1">Tractability results for the Double-Cut-and-Join circular median problem</a>. 
<br><b>A. Mahmoody-Ghaidary, C. Chauve, L. Stacho. </b> <a href="http://arxiv.org/abs/1111.5872">arXiv:cs.DM/11115872v1</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CPM11.pdf">Tractability results for the Consecutive-Ones  Property with multiplicity</a>. 
<br><b>C. Chauve, J. Manuch, M. Patterson, R. Wittler.</b> <a href="cpm2011.unipa.it">CPM 2011</a>, <a href="http://www.springerlink.com/content/w8227502h51r644t/">LNCS 6661:90-103</a>.

<li><a style="color: #800000" href="http://dx.doi.org/10.1016/j.tcs.2010.11.020">A new algorithm for aligning nested arc-annotated sequences under arbitrary weight schemes</a>. 
<br><b>A. Ouangraoua, V. Guignon, S. Hamel, C. Chauve.</b> <a href="http://dx.doi.org/10.1016/j.tcs.2010.11.020">Theoret. Comput. Sci. 412:753-764</a>. 
</ul>

<h1>2010</h1>
<ul>
<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//cmb.2010.0092.pdf"> Yeast ancestral genome reconstructions: the possibilities of computational methods II</a>. 
<br><b>C. Chauve, H. Gavranovic, A. Ouangraoua, E. Tannier.</b> <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2010.0092">J. Comput. Biol. 17:1097-1112</a>. 

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//cmb.2010.0113.pdf">Minimal
  Conflicting Sets for the Consecutive Ones Property in ancestral genome reconstruction</a>. 
  <br><b>C. Chauve, U.-U. Haus, T. Stephen, V.P. You. </B> <a
  href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2010.0113"> J. Comput. Biol. 17:1167-1181</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CPM2010.pdf">Breakpoint distance and PQ-trees</a>. 
<br><b>H. Jiang, C. Chauve, B. Zhu.</b> 
<a href="http://cs.nyu.edu/parida/CPM2010/">CPM 2010</a>,  <a href="http://www.springerlink.com/content/7ph132422r0q0761">LNCS 6219:112-124</a>.
</ul>

<h1>2009</h1>
<ul>
<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//ISBRA09.pdf">Prediction of contiguous ancestral regions in the amniote ancestral genome</a>. 
<br><b>A. Ouangraoua, F. Boyer, A. McPherson, E. Tannier, C. Chauve. </b> <a href="http://www.cs.gsu.edu/isbra09/">ISBRA 2009</a>, <a href="http://dx.doi.org/10.1007/978-3-642-01551-9_18">LNCS/LNBI 5542:173-185</a>.

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//JCB_RCG08_2.pdf">Space of Gene/Species Trees Reconciliations and 
Parsimonious Models</a>.
<br><b>J.P. Dpyon, S. Hamel, C. Chauve. </b> <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2009.0095">J. Comput. Biol. 16:1399-1418</a>. 

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//JCB_RCG08_1.pdf">Computation of perfect DCJ rearrangement scenarios 
with linear and circular chromosomes</a>.
<br><b>S. B&eacute;rard, A. Chateau, C. Chauve, C. Paul, E. Tannier.</b> <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2009.0088">J. Comput. Biol. 16:1287-1309</a>. 

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//RECOMB09.pdf">New perspectives on gene family
evolution: losses in reconciliation and a link with supertrees</a>. 
<br><b>C. Chauve, N. El-Mabrouk.</b> <a href="http://www.bio5.org/recomb2009/">RECOMB 2009</a>, <a href="http://dx.doi.org/10.1007/978-3-642-02008-7_4">LNCS/LNBI 5541:46-58</a>. 
</ul>

<h1>2008</h1>

<ul>
<li><a style="color: #800000"
href="http://www.cecm.sfu.ca/~cchauve/Publications//PLOSCB4.pdf">A methodological framework for the
reconstruction of contiguous regions of ancestral genomes and its
application to mammalian
genomes</a>. 
<br><b>C. Chauve, E. Tannier. </b> <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1000234;jsessionid=A8E1CCB01BBC1E4D167F20E51A44195A">PLoS Comput. Biol. 4:e1000234</a>. 

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//JCB-RCG07.pdf">Gene family evolution by duplication, speciation and loss</a>. 
<br><b>C. Chauve, J.P. Doyon, N. El-Mabrouk.</b> <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2008.0054">J. Comput. Biol. 15:1043-1062</a>.

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//IPL106.pdf">A more efficient algorithm for perfect sorting by reversals</a>. 
<br><b>S. B&eacute;rard, C. Chauve, C. Paul. </b> <a href="http://dx.doi.org/10.1016/j.ipl.2007.10.012">Inform. Proc. Letters 106:90-95</a>.

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CLUSTERS-WILEY07.ps">Formal models of gene clusters</a>.
<br><b>A. Bergeron, C. Chauve, Y. Gingras.</b> <a href="http://ca.wiley.com/WileyCDA/WileyTitle/productCd-0470097736.html">Bioinformatics Algorithms: Techniques and Applications, pp. 177-202</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//SIAMJDM-08.pdf">Computing common intervals of K permutations with application to graph decomposition</a>. 
<br><b>A. Bergeron, C. Chauve, F. de Montgolfier, M. Raffinot.</b> <a href="http://scitation.aip.org/getabs/servlet/GetabsServlet?prog=normal&id=SJDMEC000022000003001022000001&idtype=cvips&gifs=Yes">SIAM J. Discrete Math. 22:1022-1039</a>.
</ul>

<h1>2007</h1>
<ul>
<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//TCBB-WABI05.pdf">Perfect sorting by reversals is not always difficult</a>.
<br><b>S. B&eacute;rard, A. Bergeron, C. Chauve, C. Paul. </b> <a href="http://doi.ieeecomputersociety.org/10.1109/TCBB.2007.1011">IEEE/ACM TCBB 4:4-16</a>. 

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//TCBB-IWBRA06.pdf">Comparing genomes with duplicates: a computational complexity point of view</a> .
<br><b>G. Blin, C. Chauve, G. Fertin, R. Rizzi, S. Vialette. </b> <a href=" http://doi.ieeecomputersociety.org/10.1109/TCBB.2007.1069">IEEE/ACM TCBB 4:523-534</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//APBC07.pdf">
Exploring genome rearrangements using virtual hybridization</a>. 
<br><b>M. Belacid, A. Bergeron, C. Chauve, G. Poisson, M. Vendette. </b> <a href="http://www.cs.hku.hk/apbc2007">APBC 2007</a>, <a href="http://www.icpress.co.uk/lifesci/p501.html">Adv. Bioinfo. Comput. Biol. 5:205-214</a>.

<li><a style="color: #800000" href="http://dx.doi.org/10.1016/S1672-0229(07)60022-9">FragAnchor a large-scale all Eukaryota predictor of glycosylphosphatidylinositol-anchor in protein sequences by qualitative scoring</a>.
<br><b>G. Poisson, X. Chen, C. Chauve, A. Bergeron. </b> <a href="http://dx.doi.org/10.1016/S1672-0229(07)60022-9">Genomics, Proteomics & Bioinformatics 5:121-130</a>.
</ul>

<h1>2006</h1>
<ul>
<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CG06.pdf">Inferring positional homologs using common intervals of sequences</a>. 
<br><b>G. Blin, A. Chateau, C. Chauve, Y. Gingras. </b> <a href="http://www.crm.umontreal.ca/Genomics06/">RCG 2006</a>, <a href="http://dx.doi.org/10.1007/11864127_3">LNCS/LNBI 4205:24-38</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//RR-Bielefeld-06.pdf">
On common intervals with errors</a>.
<br><b>C. Chauve, Y. Diekman, S. Heber, J. Mixtacki, S. Rahmann, J. Stoye. </b> Report 2006-02, <a href="http://www.techfak.uni-bielefeld.de/data/eng/index.html">Technische Fakultat der Universitat Bielefeld</a>. 
</ul>


<h1>2005</h1>

<ul>
<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//SPIRE05.ps">An edit distance between RNA stem-loops</a>.
<br><b>V. Guignon, C. Chauve, S. Hamel. </b> <a href="http://www.la-web.org/spire2005">SPIRE
2005</a>, <a href="http://dx.doi.org/10.1007/11575832_38">LNCS
3772:334-345</a>. 

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CG05.ps">Genes order and
phylogenetic reconstruction: application to gamma-Proteobacteria</a>.
<br><b>G. Blin, C. Chauve, G. Fertin. </b> <a href="http://www.gen.tcd.ie/recomb_cg">RCG
2005</a>, <a href="http://dx.doi.org/10.1007/11554714_2">LNCS/LNBI 3678:11-20</a>.
</ul>

<h1>2004</h1>
<ul>
<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//WABI04.ps">Reconstructing ancestral gene orders using conserved intervals</a>. 
<br><b>A. Bergeron, M. Blanchette, A. Chateau, C. Chauve. </b> <a  href="http://www.ii.uib.no/algo2004/wabi2004/">WABI 2004</a>, <a  href="http://dx.doi.org/10.1007/978-3-540-30219-3_2">LNCS/LNBI 3240:14-25</a>. 

<li><a  style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CG04.ps">Conserved structures in evolution scenarios</a>.
<br><b>S. B&eacute;rard, A. Bergeron, C. Chauve. </b> <a href="http://www.nada.kth.se/~jensl/recombcg2004.html">RCG 2004</a>,
<a href="http://dx.doi.org/10.1007/978-3-540-32290-0_1">LNCS/LNBI 3388:1-15</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//CompBioNets04.ps">The breakpoints distance for signed sequences</a>.
<br><b>G. Blin, C. Chauve, G. Fertin. </b> <a href="http://compbionets.cin.ufpe.br/welcome.html">CompBioNets 2004</a>,
<a href="http://www.collegepublications.co.uk/algorithmics/?00003">Texts in Algorithms 3:3-16</a>.

<li><a style="color: #800000" href="http://www.cecm.sfu.ca/~cchauve/Publications//TCS326.ps">On maximal instances for the original syntenic distance</a>.
<br>C. Chauve, G. Fertin. </b> <a href="http://dx.doi.org/10.1016/j.tcs.2004.05.006">Theoret. Comput. Sci. 326:29-43</a>. 
</ul>

<h1>2002</h1>
<ul>
<li><a style="color: #800000" name="jobim02" href="http://www.cecm.sfu.ca/~cchauve/Publications//JOBIM02.ps">
On the properties of sequences of reversals that sort a signed permutation</a>.
<br><b>A. Bergeron, C. Chauve, T. Hartman, K. St-Onge. </b> <a href="http://www.irisa.fr/manifestations/2002/jobim/">JOBIM 2002</a>, pp. 99-108. 2002.
</ul>








</div>
</body>
</html>
