<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html hola_ext_inject="ready" xmlns="http://www.w3.org/1999/xhtml"><head>
<meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<meta content="Computational paleogenomics Lab at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body hola-ext-player="1">
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
Computational Methods for Paleogenomics and Comparative Genomics
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">
<a href="index.html">HOME</a> -
RESEARCH -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
<a href="software.html">SOFTWARE AND DATA</a> -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

<hr>
 
<h2>Algorithms for NGS data analysis</h2>
In a fruitful collaboration with Faraz Hach (UBC and Vacouver Prostate Center)  
we are developing novel and efficient algorithms for processing large genomic and transcriptomic sequence
data sets, mostly motivated by questions from cancer genomics. We are currently focusing on the analysis of
Third Generation Sequencing data (PacBio and Nanopore) 
[<a href="https://www.biorxiv.org/content/10.1101/2021.01.20.427493v1.full.pdf">Freddie (2021)</a>,
<a href="https://www.cell.com/iscience/pdf/S2589-0042(20)30577-0.pdf">HASLR (2020)</a>,
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27587673">CoLoRMap (2016)</a>].
We are also working on data from other sequencing protocols such as barcoded short reads
[<a href="https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/bty888/26145501/bty888.pdf">Calib (2019)</a>]
and linked reads (collaboration with Rayan Chikhi)
[<a href="https://drops.dagstuhl.de/opus/volltexte/2020/12800/pdf/LIPIcs-WABI-2020-11.pdf">WABI 2020</a>].

<h2>Genome evolution</h2> 
One of our favorite research problems, historically the initial research of our group, 
is the reconstruction of ancestral genome structures (genome maps based on synteny blocks, or
ancestral gene orders) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29277873">MMB 2018</a>]. 
We work within two methodological frameworks for this problem: a local approach, that
considers a single ancestral genome within a given species phylogeny 
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19043541">PLoS Comput Biol (2008)</a>,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/22820205">ANGES (2012)</a>],
and a global (aka small parsimony approach), that considers all
ancestral genomes of a species phylogeny at once 
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/28166504">PhySca (2017)</a>].

Over the last few years, we aimed to extend these approaches in order to work within a 
model accounting for  gene family events such as gene duplication,
loss or transfer
[<a href="https://almob.biomedcentral.com/track/pdf/10.1186/s13015-020-00169-y">SCJ-SGD (2020)</a>,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/28402423">DeCoSTAR (2017)</a>]. 
This line of work also motivated a series of papers on the reconciliation between gene trees and species
trees [<a href="https://www.ncbi.nlm.nih.gov/pubmed/27153713">ecceTERA (2016)</a>,
 <a href="https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7959594">SuGeT</a>].

<h2>Anopheles mosquito genomics and comparative scaffolding</h2>  
We did apply our comparative genomics and genome rearrangement algorithms to 
a fascinating, large-scale, data set composed of toughly twenty <i>Anopheles</i> mosquito genomes 
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/25554792">Science (2015)</a>]. 
This in turn raises interesting questions on how to handle fragmented genome assemblies in genome 
rearrangement studies [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26450761">ArtDeCo (2015)</a>,
<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-018-4466-7.pdf">ADSeq (2018)</a>,
<a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0728-3">BMC Biology (2020)</a>].
  
<h2>Pathogen genomics</h2> 
We recently started a very active collaboration with the SFU Computational Epidemiology lab of 
<a href="http://www.sfu.ca/~leonid">Dr. Leonid Chindelevitch</a>, 
and Dr. Will Hsiao (BC center for Disease Control), focusing on the development and application of 
novel bioinformatics tools for the analysis of whole-genome sequencing data of microbial pathogens
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/29319471">MentaLiST (2018)</a>,
 <a href="https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/btz413/28694516/btz413.pdf">HyAsP (2019)</a>,
 <a href="https://link.springer.com/content/pdf/10.1007%2F978-3-030-42266-0_9.pdf">PathoGiST (2020)</a>].

<h2>Ancient DNA</h2>
The recent breakthroughs in ancient DNA (aDNA) sequencing naturally complements
complement methods for reconstructing ancient genomes. This motivated our project to
assemble recently sequenced historical samples of the pathogen <i>Yersinia pestis</i>,
showing that it is possible to go beyond single nucleotide mutations to analyze ancient 
pathogens data
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/24068034">FPSAC (2013)</a>,
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29114402">AGaPES (2017)</a>].
 
<h2>Big data flow cytometry bioinformatics</h2>
This topic stems from a collaboration with Ryan Brinkman (BC Cancer Agency) and Max Libbrecht 
(SFU), funded by Genome Canada and NIH
 [<a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/16/837765.full.pdf">flowGraph (2019)</a>,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/29462241">flowLearn (2018)</a>].

<h2>Combinatorics problems motivated by bioinformatics questions</h2>
We are also interested in applications of enumerative combinatorics techniques to theoretical questions motivated by 
bioinformatics problems such as RNA secondary structures alignment [<a href="https://www.worldscientific.com/doi/pdf/10.1142/S0129054118420030">IJFCS 2018</a>],
RNA design [<a href="https://dl.acm.org/ft_gateway.cfm?id=3342163&ftid=2082503&dwn=1&CFID=158103347&CFTOKEN=c7798d30cdfe3edf-A0C164A2-C903-049C-E1728873D24A4D65">BCB 2019</a>],
gene trees counting [<a href="https://doi.org/10.1007/s00285-019-01465-x">JMB 2020</a>], and sequence alignment
[<a href="https://cchauve.github.io/PSC_2021_300621.pdf">PSC 2021</a>]. 


</body></html>

