<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational paleogenomics Lab at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<img src="paleo_fichiers/irmacs.jpg" alt="" width="70px">
Computational Methods for Paleogenomics and Comparative Genomics
<img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">
<a href="index.html">HOME</a> -
<a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
SOFTWARE AND DATA -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

This page contains software developed in our group at Simon Fraser
University, as well as links to software and websites developed in
collaboration with other groups.
  
<hr><b>Github.</b>
The most recent versions of the software developed in our lab and with our collaborators are available from our
<a href="https://github.com/cchauve">GitHub repository</a>. The main tools available there and on some of our collaborators github sites:
<ul>
  <li><b>Pathogens genomics</b>
  <ul>
  <li>MentaLiST, frugal and accurate Multi-Locus Sequence Typing: <a href="https://github.com/WGS-TB/MentaLiST">MentaLiST</a>.
  <li>PRINCE, calling Copy-Number Variants from NGS data: <a href="https://github.com/WGS-TB/PythonPRINCE">PRINCE</a>.
  <li>HyAsP, a hybrid assembler for plasmids: <a href="https://github.com/cchauve/HyAsP">HyAsP</a>.
  </ul>
  <li><b>NGS tools</b>
  <ul>
  <li>CoLoRMap, hybrid long reads correction: <a href="https://github.com/sfu-compbio/colormap">CoLoRMap</a>.
  <li>LRCstats, evaluating long reads correction tools: <a href="https://github.com/cchauve/lrcstats">LRCstats</a>.
  <li>Calib, clustering UMI-barcoded sequencing data: <a href="https://github.com/vpc-ccg/calib">Calib</a>.
  </ul>
  <li><b>Comparative genomics</b>
  <ul>
  <li>ANGeS, reconstructing ancestral genome maps: <a href="https://github.com/cchauve/ANGeS">ANGeS</a>.
  <li>SCJ-TD-FD, the Single-Cut-or-Join distance with duplicated genes: <a href="https://github.com/cchauve/SCJ-TD-FD">SCJ-TD-FD</a>.
  <li>PhySca, the Single-Cut-and-Join (SCJ) small parsimony problem for weighted gene adjacencies: <a href="https://github.com/cchauve/PhySca">PhySca</a>.
  <li>DeCoStar, reconstructing parsimonious histories of gene adjacencies: <a href="https://github.com/WandrilleD/DeCoSTAR">DeCoStar</a> (see also <a href="https://github.com/YoannAnselmetti/DeCoSTAR_pipeline">DeCoSTAR_pipeline</a> for an application to comparative scaffolding).
  <li>DeClone, Evolutionary scenarios for gene adjacencies in a Gibbs-Boltzmann framework : <a href="https://github.com/cchauve/DeClone">DeClone</a>.
  <li>ecceTERA, gene tree species tree reconciliation: <a href="http://mbb.univ-montp2.fr/MBB/download_sources/16__ecceTERA">ecceTERA</a>
  </ul>
  <li><b>Paleogenomics and ancient DNA</b>
  <ul>
  <li>FPSAC, fast phylogenetic scaffolding of ancient contigs: <a href="https://github.com/cchauve/FPSAC">FPSAC</a>.
  <li>AGaPeS, comparative gap filling in ancient DNA assemblies: <a href="https://github.com/cchauve/AGapEs">AGapEs</a>.
  <li>EWRA, Edge-Weighted Reconstruction integrating aDNA Assembly graph: <a href="https://github.com/nluhmann/EWRA">EWRA</a>.
  </ul>
  <li><b>Other</b>
  <ul>
  <li>flowLearn, a machine-learning algorithm for gating flow cytometry data: <a href="https://github.com/mlux86/flowLearn">flowLearn</a>.
  <li>RNA-unchained, Chaining sequence/structure seeds for computing RNA similarity: <a href="https://github.com/cchauve/RNA-unchained">RNA-unchained</a>.
  </ul>
</ul>

<hr><b>Older stuff.</b><hr>

<p><hr>
<b>FragAnchor</b>. GPI-Anchored Protein Prediction.

<br>Website: <a href="http://navet.ics.hawaii.edu/~fraganchor/NNHMM/NNHMM.html">FragAnchor</a>.
<br>References: <a href="http://dx.doi.org/10.1016/S1672-0229(07)60022-9">Paper</a>.

<p><hr>
<b>splitMixed</b>. Finding in a matched tumor/normal paired-end reads dataset a consistent explanation for a (mixed) tumor-subset of deletions.
<br><a href="http://www.biomedcentral.com/content/supplementary/1471-2105-12-s9-s21-s1.zip">Code</a>.
<br>Author: <a href="http://www.cebitec.uni-bielefeld.de/~roland">Roland Wittler</a>. 
<br>References: <a href="http://www.biomedcentral.com/1471-2105/12/S9/S21">Paper</a>.

<p><hr>
<b>BRASERO</b>. Benchmarking RNA Secondary Structure Comparison Algorithms.

<br>Website: <a href="http://brasero.labri.fr/">BRASERO</a>.
<br>Author: <a href="http://www.labri.fr/perso/allali/">Julien Allali</a>.
<br>References, <a href="http://www.cecm.sfu.ca/~cchauve/Publications/JOBIM08-77-poster.pdf">Poster</a>, <a href="http://www.hindawi.com/journals/abi/2012/893048/">Paper</a>.

<p><hr>
<b>Korak</b>. Probabilistic exploration of the space of gene tree/species tree reconciliations.

<br><a href="KORAK/">Code and data</a>.
<br>Author: <a href="http://www.lirmm.fr/~doyon/">Jean-Philippe Doyon</a>.
<br>References: <a href="http://www.computer.org/portal/web/csdl/doi/10.1109/TCBB.2011.64">Paper</a>.

<br>References: <a href="http://www.cecm.sfu.ca/~cchauve/Publications/PHYLOGENOMICS06-b.pdf">Poster</a>, 
<a href="http://www.cecm.sfu.ca/~cchauve/Publications/CG06.pdf">Paper</a>.

<p><hr>
<b>RNA StrAT</b>. RNA Structure Analysis Toolkit

<br>Website: <a href="http://mr-nutz.no-ip.org/rnastrat/?p=tools">RNA StrAT</a>
<br>Author: <a href="http://www.bioversityinternational.org/about-us/who-we-are/staff-bios/single-details-bios/guignon-valentin/">Valentin Guignon</a>.
<br>References: <a href="http://www.cecm.sfu.ca/~cchauve/Publications/ISMB08-D31-poster.pdf">Poster</a>. 


</div>
</body>
</html>
