<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta content="Computational paleogenomics Lab at SFU" name="description">
<meta name="keywords" content="Cedric Chauve, SFU">
<meta name="author" content="Cedric Chauve">

<title>Computational Methods for  Comparative Genomics at Simon Fraser University</title>
<link rel="stylesheet" href="paleo_fichiers/mystyle.css" type="text/css" media="screen">
</head>

<body>
<div class="hd">

<div style="clear: both; float: left;">
<h1>
<!img src="paleo_fichiers/SFU.jpg" alt="" width="104px">
Computational Methods for Paleogenomics and Comparative Genomics
</h1>
</div>

<div style="float: right;">
</div>

</div>

<div style="clear: both; float: left;" id="navbar">
<a href="index.html">HOME</a> -
<a href="research.html">RESEARCH</a> -
<a href="people.html">PEOPLE</a> -
<a href="publications.html">PUBLICATIONS</a> -
SOFTWARE AND DATA -
<a href="support.html">SUPPORT AND COLLABORATIONS</a>
</div>

<div class="hd">

This page contains software developed in our group at Simon Fraser
University, as well as links to software and websites developed in
collaboration with other groups.
  
<hr><b>Github.</b>
The most recent versions of the software developed in our lab and with our collaborators are available from our
<a href="https://github.com/cchauve">GitHub repository</a>. The main tools available there and on some of our collaborators github sites:
<ul>
  <li><b>Plasmids bioinformatcs</b>
  <ul>
  <li><a href="https://github.com/cchauve/PlasBin-flow">PlasBin-flow</a>, optimization-based binning of plasmid contigs from assembly graphs.
  <li><a href="https://github.com/cchauve/plASgraph2">plASgraph2</a>, detection of plasmid contigs in assembly graphs.
  <li><a href="https://github.com/cchauve/PlasEval">PlasEval</a>, evaluating plasmids binning methods.
  <li><a href="https://github.com/cchauve/HyAsP">HyAsP</a>, hybrid binning of plasmid contigs from assembly graphs.
  </ul>
  <li><b>Pathogens genomics</b>
  <ul>
  <li><a href="https://github.com/WGS-TB/MentaLiST">MentaLiST</a>, whole-genome and core-genome multi-locus sequence typing (MLST).
  <li><a href="https://github.com/WGS-TB/PythonPRINCE">PRINCE</a>, estimatig the copy number of VNTR.
  <li><a href="https://github.com/WGS-TB/PathOGiST">PathoGiST</a>, clustering pathogen isolates by combining multiple genotyping signals.
  </ul>
  <li><b>NGS tools</b>
  <ul>
  <li><a href="https://github.com/vpc-ccg/tksm">TKSM</a>, highly modular, user-customizable, and scalable transcriptomic sequencing long-read simulator.
  <li><a href="https://github.com/vpc-ccg/scTagger>scTagger</a>, fast and accurate matching of cellular barcodes across short- and long-reads of single-cell RNA-seq experiments.
  <li><a href="https://github.com/vpc-ccg/freddie/">Freddie</a>, annotation-independent detection and discovery of transcriptomic alternative splicing isoforms using long-reads.
  <li><a href="https://bitbucket.org/f0t1h/genion">Genion</a>, detecting gene fusions from transcriptomics long reads.
  <li><a href="https://github.com/vpc-ccg/haslr">HASLR</a>, fast hybrid assembly of long reads.
  <li><a href="https://github.com/sfu-compbio/colormap">CoLoRMap</a>, hybrid correction of long reads.
  <li><a href="https://github.com/cchauve/lrcstats">LRCstats</a>, evaluating long-reads error-correction methods.
  <li><a href="https://github.com/vpc-ccg/calib">Calib</a>, alignment-free clustering of UMI tagged DNA molecules.
  </ul>
  <li><b>Comparative genomics</b>
  <ul>
  <li><a href="https://github.com/cchauve/AGO-pipeline">AGO</a>, a pipeline for ancestral gene orders reconstruction.
  <li><a href="https://github.com/cchauve/ANGeS">ANGeS</a>, reconstructio of ancestral genomes maps.
  <li><a href="https://github.com/danydoerr/spp_dcj">SPP-DCJ</a>, the small parsimony problem with the Single-Cut-or-Join distance.
  <li><a href="https://github.com/cchauve/SCJ-with-SGD">SCJ-with-SGD</a>, the directed Single-Cut-or-Join distance with duplicated genes.
  <li><a href="https://github.com/cchauve/PhySca">PhySca</a>, Single-Cut-or-Join small-parsimony for weighted gene adjacncies.
  <li><a href="https://github.com/WandrilleD/DeCoSTAR">DeCoStar</a>, reconstructing the ancestral organization of genes or genomes using reconciled phylogenies .
  <li><a href="https://github.com/cchauve/DeClone">DeClone</a>, Eovlution of genes neighborhood within reconciled phylogenies: an ensemble approach.
  <li><a href="https://github.com/YoannAnselmetti/ADseq-Anopheles-APBC2018">ADseq</a>, joint scaffolding extant and ancestral genomes.
  <li><a href="http://mbb.univ-montp2.fr/MBB/download_sources/16__ecceTERA">ecceTERA</a>, gene tre/species tree reconcliation.
  <li><a href="https://github.com/ssolo/ALE/tree/master/maxtic">MaxTiC</a>, ranking of a phylogenetic tree by maximum time consistency with lateral gene transfers.
  </ul>
  <li><b>Paleogenomics and ancient DNA</b>
  <ul>
  <li><a href="https://github.com/cchauve/FPSAC">FPSAC</a>, fast phylogenetic scaffolding of ancient contigs.
  <li><a href="https://github.com/cchauve/AGapEs">AGapEs</a>, comparative scaffolding and gap filling of ancient bacterial genomes.
  <li><a href="https://github.com/nluhmann/EWRA">EWRA</a>, joint scaffolding of ancient cotigs and ancestral genome reconstruction.
  </ul>
  <li><b>Other</b>
  <ul>
  <li><a href="https://github.com/cchauve/CEDAR">CEDAR</a>, atools for manipluating a vector representation for phylogenetic trees.
  <li><a href="http://github.com/aya49/flowGraph">flowGraph</a>, automated identification of maximal differential cell populations in flow cytometry data.
  <li><a href="https://github.com/mlux86/flowLearn">flowLearn</a>, a machine-learning tool for gating flow cytometry data.
  <li><a href="https://github.com/cchauve/RNA-unchained">RNA-unchained</a>, Chaining sequence/structure seeds for computing RNA similarity.
  <li><a href="https://github.com/cchauve/DLTcount">DLTCount</a>, counting and sampling gene family histories. 
  </ul>
</ul>

<hr><b>Older stuff.</b><hr>

<p><hr>
<b>FragAnchor</b>. GPI-Anchored Protein Prediction.

<br>Website: <a href="http://navet.ics.hawaii.edu/~fraganchor/NNHMM/NNHMM.html">FragAnchor</a>.
<br>References: <a href="http://dx.doi.org/10.1016/S1672-0229(07)60022-9">Paper</a>.

<p><hr>
<b>Amniotes ancestral genome</b>. reconstructing the gene order of the amniote ancestral genome/.
<br><a href="AMNIOTES/btr461_code.tgz">Code and data</a>
<br>Reference, <a href="https://doi.org/10.1093/bioinformatics/btr461">Reconstructing the architecture of the ancestral amniote genome</a>
  
<p><hr>
<b>splitMixed</b>. Finding in a matched tumor/normal paired-end reads dataset a consistent explanation for a (mixed) tumor-subset of deletions.
<br><a href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-S9-S21/MediaObjects/12859_2011_4828_MOESM1_ESM.zip">Code</a>.
<br>Author: <a href="http://www.cebitec.uni-bielefeld.de/~roland">Roland Wittler</a>. 
<br>References: <a href="http://www.biomedcentral.com/1471-2105/12/S9/S21">Paper</a>, <a href="http://www.cecm.sfu.ca/~cchauve/SUPP/AMNIOTE2010/">Supp. Mat.</a>.

<p><hr>
<b>BRASERO</b>. Benchmarking RNA Secondary Structure Comparison Algorithms.

<br>Website: <a href="http://brasero.labri.fr/">BRASERO</a>.
<br>Author: <a href="http://www.labri.fr/perso/allali/">Julien Allali</a>.
<br>References, <a href="http://www.cecm.sfu.ca/~cchauve/Publications/JOBIM08-77-poster.pdf">Poster</a>, <a href="http://www.hindawi.com/journals/abi/2012/893048/">Paper</a>.

<p><hr>
<b>Korak</b>. Probabilistic exploration of the space of gene tree/species tree reconciliations.

<br><a href="KORAK/">Code and data</a>.
<br>Author: <a href="http://www.lirmm.fr/~doyon/">Jean-Philippe Doyon</a>.
<br>References: <a href="http://www.computer.org/portal/web/csdl/doi/10.1109/TCBB.2011.64">Paper</a>.

<br>References: <a href="http://www.cecm.sfu.ca/~cchauve/Publications/PHYLOGENOMICS06-b.pdf">Poster</a>, 
<a href="http://www.cecm.sfu.ca/~cchauve/Publications/CG06.pdf">Paper</a>.

</div>
</body>
</html>
